libraries/lib-datahandler/src/main/thrift/licenses.thrift

```diff
@@ -86,9 +87,13 @@ struct License {
      9: optional map<string, string> externalIds,
 
     // Additional informations
-	10: optional bool GPLv2Compat,
-	11: optional bool GPLv3Compat,
+	// 10: optional bool GPLv2Compat,
+	// 11: optional bool GPLv3Compat,
 	12: optional string reviewdate,
+
+	15: optional Ternary GPLv2Compat = sw360.Ternary.UNDEFINED,
```

Comment:
You believe you can skip the qualifying `sw360.` and just write `Ternary.UNDEFINED`